src/components/wishlist/Wishlist.jsx

```diff
@@ -1,17 +1,22 @@
 import "components/wishlist/wishlist.css";
-// import { useCart } from "context/cart-context";
+import { useCart } from "context/cart-context";
 import { useWishlist } from "context/wishlist-context";
 import { BsSuitHeart, BsSuitHeartFill } from "react-icons/bs";
 
 const Wishlist = () => {
   const { wishlistState, wishlistDispatch } = useWishlist();
   const { wishlist } = wishlistState;
```

Comment:
` const { wishlistState:{wishlist}, wishlistDispatch } = useWishlist();`
You can do this also instead of two lines. It is a personal preference though

Reply:
Yeah, that would be better